### DISCUSSION TITLE
--tags syntax for aws?

### DISCUSSION THREAD

**Discussion by ranshe** [cycloidio/terracognita#321] [Q&A]
I'm trying to filter resources by tags.
Tried multiple variations of "terracognita aws password_stuff --hcl out.hcl --include list --tags key:value", with ', " or nothing surrounding each of key:value and both of them.
The run always looks the same:
Importing with filters:
	Tags:    [], <-- tags not applied
	Include: [whatever_was_listed],
	Exclude: [],
	Targets: [],

What am I doing wrong?

**Comment by xescugc** [comment]
There is a PR to solve this already :) https://github.com/cycloidio/terracognita/pull/323.

You are doing nothing wrong :)

**Reply by giuseppe-lettere** [reply]
Thaks a Lot in Advance :-) is the PR not approved yet ?
